README.md

```diff
@@ -15,14 +15,21 @@ It comes with a lot of performance improvements, and it brings minimalistic widg
 If you want to support this project, you can [click here](https://paypal.me/pigigaldi)!
 Pock will always be available as free software!
 
+## Install
+Using brew:
+```
+brew cask install pock
+```
 
+Manually:
+- Go to the [latest release](https://github.com/pigigaldi/Pock/releases) and download the zip file.

```

Comment:
Can you please reference Pock website for downloading the latest release?

Reply:
@pigigaldi should be fixed.now